### DISCUSSION TITLE
Not seeing database in PostgreSQL V14 with V13 also installed

### DISCUSSION THREAD

**Discussion by fredatgithub** [dbeaver/dbeaver#14425] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: Windows10 20H2 19042.1288
- DBeaver version:  21.2.3.202110151816
- Additional extensions: none

#### Connection specification:
- Database name and version: CryptoCurrencies V14 PostgreSQL JDBC Driver
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? NO

#### Describe the problem you're observing:
I don't see my database in the database view tab
#### Steps to reproduce, if exist:
Create a new database in PostgreSQL V14 with PostgreSQL V13 installed on the same machine. Then refresh Database list by pressing F5 in Database view tab;

See screenshot for details

![bug-dbeaver-21 2 3](https://user-images.githubusercontent.com/6720055/139120518-ee93d3ec-19c3-4af9-bb8d-770480e764c2.JPG)
.

**Comment by ShadelessFox** [comment, marked as answer]
Hello @fredatgithub,

Perhaps you don't have this option enabled?
![java_3OPH4jDygQ](https://user-images.githubusercontent.com/35821147/139120834-5b3885a9-5f37-45e5-86cf-8e8590e31608.png)

**Comment by fredatgithub** [comment]
Thanks a lot, I didn't know
it is fine now
